CMakeLists.txt.in

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 2.8.11)
+
+project(googletest-download NONE)
```

Comment:
Why do we not rely on `ExternalProject` to download the source?

Reply:
That's how we did it in Cajita originally but @junghans suggested we use `FetchContent` instead